js/classes/entities/midground/drops/ammo_drop.js

```diff
@@ -10,7 +10,7 @@ class AmmoDrop {
      * @param {Vector} pos The position at which the AmmoDrop should start.
      * @param {number} type The type of AmmoDrop that should be generated. AmmoDrop.ROCK, .BOMB, etc.
      */
-    constructor(pos, type, amount = 1, hasGravity = true, popInAir = false) {
+    constructor(pos, type, amount = 1, hasGravity = true, popInAir = true) {
```

Comment:
What is `popInAir` for? It doesn't seem to be used. Same thing in `FoodDrop` and `RuneDrop`.

Reply:
`popInAir` is checked later on in the constructor to tell if the AmmoDrop should have an upwards y velocity on instantiation.

Reply:
Oh you're right, I'm blind and didn't realize you only changed the default value lol. Sorry about that!